glean/src/core/pings/maker.ts

```diff
@@ -223,15 +223,25 @@ export async function collectAndStorePing(identifier: string, ping: PingType, re
     return;
   }
 
+  let modifiedPayload;
+  try {
+    modifiedPayload = await CoreEvents.afterPingCollection.trigger(collectedPayload);
+  } catch(e) {
+    console.error(
+      `Error while attempting to modify ping payload for the "${ping.name}" ping using`,
+      `the ${JSON.stringify(CoreEvents.afterPingCollection.registeredPluginIdentifier)} plugin.`,
+      "Ping will not be submitted. See more logs below.\n\n",
```

Comment:
```suggestion
      "Ping will not be submitted. See more logs below.\n",
```

Why did I add two line breaks...